tests/tests.yaml

```diff
@@ -140,5 +140,6 @@ answer_tests:
 other_tests:
   unittests:
      - '--exclude=test_mesh_slices'  # disable randomly failing test
+     - '--exclude=test_load_from_path'  # PY2
```

Comment:
oops, it was disabled for py3 too. It's apparently case sensitive, it should be "py2". Sorry for the wrong example.

Reply:
Oh, nice catch !